### PR TITLE
solution

### PR DESCRIPTION
**Pull request by lukacsja**
[DEMO LINK](https://lukacsja.github.io/react_keyboard/)

### CODE REVIEW

**Review by mmahfuzi — APPROVED** [review]
Good job :)💯

**Comment by lukacsja** [comment]
> Good job :)💯

thank you again for making it possible 😄

**Comment by mmahfuzi** [comment]
> > Good job :)💯
> 
> thank you again for making it possible 😄

Welcome, Thank you for bearing with me :D

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">
+          {
+            (!this.state.value)
+              ? 'Nothing was pressed yet'
+              : `The last pressed key is [${this.state.value}]`
+          }
```
Comment (meforoma) [inline src/App.tsx:32]: the only thing we've missed is making usage of `this.state.value` shorter:
```
render () {
  const { value } = this.state;
  ...
```

**Review by meforoma — APPROVED** [review]
✅

**Pull request closed without merging**